changelog.txt

```diff
@@ -1,2 +1,6 @@
+* Fixed bug where raw body was missing on HTTPS function request
+* Fixed bug with Functions Emulator and firebase-admin@8.0.0
```

Comment:
This fixes a bug with `firebase-admin@8.0.0`, but it's still fine with everything else, right (including node 6)?

I don't know why this caught my eye the second time

Reply:
Good catch, this will subtly break 6 so I've changed this to merge into `next`